Clean package xml and set version to 0.1.0

JeroenDM opened the pull request:
Add more description and authors. Remove comments. Add nonzero version. #40

JeroenDM committed:
cleanup package.xml and add authors

JeroenDM committed:
add better description

simonschmeisser commented on package.xml:
```diff
@@ -1,53 +1,22 @@
+  <version>0.1.0</version>
+  <description>
+    <p>MoveIt kinematics plugin for industrial robots.</p>
+    <p> This plugin uses an analytical inverse kinematic library, opw_kinematics, to calculate the inverse kinematics for industrial robots with 6 degrees of freedom, two parallel joints, and a spherical wrist.</p>
```
line break?

simonschmeisser commented on package.xml:
```diff
@@ -1,53 +1,22 @@
   <!-- <url type="website">http://wiki.ros.org/opw_kinematics_plugin</url> -->
 
+  <author>Jeroen De Maeyer</author>
+  <author>Simon Schmeisser</author>
```
```suggestion
  <author>Simon Schmeisser (isys vision)</author>
```

JeroenDM committed:
Add affiliations in package.xml

Co-Authored-By: Simon Schmeisser <simon.schmeisser@isys-vision.de>

JeroenDM committed:
Add line breaks in package.xml

based on example in fanuc repo: https://github.com/ros-industrial/fanuc/blob/indigo-devel/fanuc_resources/package.xml
